pubspec.yaml

```diff
@@ -28,9 +29,12 @@ dev_dependencies:
   dart_dev: ^2.0.1
   dart_style: ^1.2.0
   dependency_validator: ^1.2.3
+  meta: ^1.1.7
   mockito: ^4.0.0
   pedantic: ^1.4.0
+  react: ^4.6.1
   test: ^1.5.1
 
 executables:
-  dart2_upgrade:
\ No newline at end of file
+  dart2_upgrade:
+  react_16_upgrade:
```

Comment:
Thoughts on removing the first underscore here and in filenames for consistency with the "dart2" codemods? `react16_upgrade`

I'm pretty sure it's my fault that it's inconsistent currently 😞

Reply:
who the hell added that?